2-ui/1-document/10-size-and-scroll-window/article.md

```diff
@@ -16,33 +16,33 @@ For instance, this button shows the height of your window:
 <button onclick="alert(document.documentElement.clientHeight)">alert(document.documentElement.clientHeight)</button>
 ```
 
-````warn header="Not `window.innerWidth/Height`"
-Browsers also support properties `window.innerWidth/innerHeight`. They look like what we want. So why not to use them instead?
+````warn header="Not `window.innerWidth/innerHeight`"
+Browsers also support properties like `window.innerWidth/innerHeight`. They look like what we want, so why not to use them instead?
 
-If there exists a scrollbar, and it occupies some space, `clientWidth/clientHeight` provide the width/height without it (subtract it). In other words, they return width/height of the visible part of the document, available for the content.
+If there exists a scrollbar, and it occupies some space, `clientWidth/clientHeight` provide the width/height without it (subtract it). In other words, they return the width/height of the visible part of the document, available for the content.
 
-...And `window.innerWidth/innerHeight` include the scrollbar.
+`window.innerWidth/innerHeight` includes the scrollbar.
 
-If there's a scrollbar, and it occupies some space, then these two lines show different values:
+If there's a scrollbar, and it occupies some space, then these two lines return different values:
```

Comment:
I would stick with "show", especially since "return" has special meaning.

Reply:
Good point. Will bring it back.

Reply:
```suggestion
If there's a scrollbar, and it occupies some space, then these two lines show different values:
```